packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -592,7 +591,7 @@ export default class Meeting extends StatelessWebexPlugin {
      * @public
      * @memberof Meeting
      */
-    this.type = null;
+    this.type = attrs.type;
```

Comment:
this seems wrong. Currently the type field is either "meeting" or "call" and it's checked against these values in many places, see for example this line here: https://github.com/webex/webex-js-sdk/blob/6ac03087c2156df9fb1f0b95b2089ac131b2f5e3/packages/node_modules/%40webex/plugin-meetings/src/meeting/index.js#L4140 
this change will now break it.
If we need to store the destination type, it should be stored in a new field with appropriate name like `destinationType`

Reply:
Changed to add new field